webnode/src/redux/services/iota.js

```diff
@@ -20,7 +20,8 @@ const checkIfClaimed = transaction => {
   const lastEpoch = moment()
     .subtract(1, "year")
     .valueOf();
-  return lastEpoch > attachedAt;
+
+  return lastEpoch < attachedAt;
```

Comment:
i think i had the logic flipped before. it should be: a sector is claimed if it was last attached after the last epoch